frontend/src/components/form/tiptap/AutoLinkDecoration.js

```diff
@@ -0,0 +1,70 @@
+import { Extension } from '@tiptap/vue-2'
+import { Plugin, PluginKey } from '@tiptap/pm/state'
+import { Decoration, DecorationSet } from '@tiptap/pm/view'
+import LinkifyIt from 'linkify-it'
+
+export const AutoLinkDecoration = Extension.create({
+  name: 'autoLinkDecoration',
+  addProseMirrorPlugins() {
+    const linkify = new LinkifyIt()
+    /**
+     * @param {import('prosemirror-model').Node} doc
+     */
+    const urlDecoration = (doc) => {
+      const decorations = []
+      doc.descendants((node, pos) => {
+        if (node.isText && linkify.pretest(node.text)) {
+          const matches = linkify.match(node.text)
+          matches?.forEach(({ index, lastIndex, url }) => {
+            if (url.charAt(0) === '/') {
+              return
+            }
+            let link
+            try {
+              link = new URL(url)
+            } catch (_) {
+              /* It can't be parsed as an url */
+            }
+            if (!link.host.includes('.') && !link.port) {
+              return
+            }
+            if (!['http:', 'https:'].includes(link.protocol)) {
+              return
+            }
+            const attrs = {
+              nodeName: 'a',
+              href: url,
+              class: 'autolink',
+              target: '_blank',
+              rel: 'noopener noreferrer',
+            }
+            if (this.editor.isEditable) {
+              attrs.onclick = `(event.metaKey || event.ctrlKey) && window.open("${link}", "_blank");`
```

Comment:
Two inputs: 
- I like the idea that you have to use `Ctrl + Click` to follow the link, but I'm unsure if all users will discover this themselves. Let's see (I would leave it like this for now)

- `Ctrl + Click` is already used by Tiptap. The text-edit dialog (bold, italic) is then displayed. Maybe you can stop this with `event.stopPropagation()`.

Reply:
> * I like the idea that you have to use Ctrl + Click to follow the link, but I'm unsure if all users will discover this themselves. Let's see (I would leave it like this for now)

Sadly the browser prevents the click if it is inside a `contenteditable`. That's why I've added the click handler. I've also made it possible to doubleclick the link to open it.

> * Ctrl + Click is already used by Tiptap. The text-edit dialog (bold, italic) is then displayed. Maybe you can stop this with event.stopPropagation().

I was able to prevent the bubble menu if you click or select within/whole link.